### PR TITLE
CLDR-16994 BRS: en_001 promotion

### PR DESCRIPTION
**Pull request by srl295**
CLDR-16994

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by srl295** [comment]
All done except the three 'maybes' - not sure what the final is on these.

File: common/annotations/en_001.xml [hunk 1]
```diff
@@ -450,5 +452,26 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
...
+				<annotation cp="👨‍⚖">judge | justice | law | man</annotation>
+		<annotation cp="👩‍⚖">judge | justice | law | woman</annotation>
+		<annotation cp="👨‍🚒">fire truck | firefighter | man | fire</annotation>
+		<annotation cp="👩‍🚒">fire truck | firefighter | woman | fire</annotation>
+		<annotation cp="🛝">amusement park | play | playground slide | theme park</annotation>
+		<annotation cp="🎡">amusement park | ferris | theme park | wheel</annotation>
+		<annotation cp="🎢">amusement park | coaster | roller | theme park</annotation>
+
+		<annotation cp="🛜">computer | internet | network | wi-fi | wifi | wireless</annotation>
+
```
Comment (srl295) [inline common/annotations/en_001.xml:475]: later CLDRModify will reorder these.

**Comment by srl295** [comment]
> build: tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestAnnotations.java#L192
(TestAnnotations.java:192) Error: : keywords for 👨‍⚖: expected java.util.HashSet<[scales, justice, man, judge]>, got com.google.common.collect.RegularImmutableSet<[judge, justice, law, man, scales]>

So… @macchiati what is this test checking?  Should en.xml not be changed here?

**Review by macchiati — CHANGES_REQUESTED** [review]
Hold off a bit until I find out about wi-fi

**Comment by srl295** [comment]
@macchiati  ping any update on `wifi`?

**Comment by srl295** [comment]
@macchiati test seems to just have hard coded values, i've updated them

**Review by macchiati — APPROVED** [review]

**Comment by srl295** [comment]
> @macchiati test seems to just have hard coded values, i've updated them

Does the golden test case here add value? If it's testing parsing, would it be better to just start with a static extract from English instead of running against live data?

**Pull request merged**